client/ayon_core/lib/transcoding.py

```diff
@@ -834,6 +834,112 @@ def get_ffprobe_streams(path_to_file, logger=None):
     return get_ffprobe_data(path_to_file, logger)["streams"]
 
 
+def get_image_info_metadata(
```

Comment:
I really don't think this is usefull. OIIO might output different data than ffprobe, and whatever is asking for the information should know if the output is from oiio or ffmpeg.

Reply:
I am only testing the function in tests. Tests parametrized for both exr and mov. And all it needs to get for the implementation is provided reliably.

Reply:
If it's used only in tests then please move it to tests, when it's added to lib it becomes maintained code.

Reply:
I see it's been used in the loader. If that's the case then please add exacmple output to the function docstring. I would also add "base" output of the function, so a developer using the function can know what the output actually should contain.

Right now with the nested dictionary manipulation and combination of OIIO and ffmpeg it is uber confusing to find out what it actually does return.

If it would always return the same data structure, it would help a lot. E.g.
```
{
     "timecode": "00:00:00",
     "frame_start": 1,
     "frame_end": 2,
}
```

Reply:
function move into [loader plugin](https://github.com/ynput/ayon-core/pull/44/commits/ede1b1737013df3b8f0a38d5d781511e9915d10c) and [tests file renamed](https://github.com/ynput/ayon-core/pull/44/commits/e485eda6110e89a70c11672f89cb96187b8fc1ad) and in more appropriate stucture.